### ISSUE TITLE
Viewport HomeKit TV

### ISSUE THREAD

**Issue by Zacklock** [hjdhjd/homebridge-unifi-protect#432]
**Is your feature request related to a problem? Please describe:**
Nope. 
Can the viewport-poe be imported to homekit as a TV with switchable inputs? 
Inputs being the viewports preset views.

**Describe the solution you'd like:**
I think this would take automations to the nest level e.g. a doorbell press changes the viewport to full screen doorbell view.

**Describe alternatives you've considered:**
Currently the only way to change the viewport's view is to login on a desktop so this would save a lot of time if you just want to change the view.

**Additional context:**

Thanks for the amazing plugin!

**Comment by hjdhjd** [comment]
Thanks for the suggestion…interesting request. I have spent no time looking at the ViewPort. I might have to pick one up and mess with it over the summer…might be a fun project to add to the backlog.

**Comment by hjdhjd** [comment]
Feature implemented in v5. Enjoy!

**Comment by Zacklock** [comment]
Thank you!!

I was able to get everything working except when I made a switch change on homekit I would get a error in the homebridge log saying the liveview could not be set untill i saw the "requires the Administrator role in order to automatically configure camera RTSP streams.".
I changed my homebridge user from Limited to Super Admin privileges and it worked first try!

Thanks again for taking the time to add this. I am going to use this so much!
